packages/editor/src/client/utility.ts

```diff
@@ -64,6 +64,31 @@ export function dateTimeLocalString(date: Date) {
   return `${year}-${month}-${day}T${hours}:${minutes}:${seconds}`
 }
 
+export function dateTimeWithSecondsLocalString(date: Date) {
```

Comment:
Nice way of handling this but the formatting libraries already do this and are a bit more flexible :)